src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/OverlappingEdgeCheck.java

```diff
@@ -184,6 +184,7 @@ private boolean isPartOfClosedWay(final Edge object)
         {
             wayIds.add(nextEdge.getIdentifier());
             final List<Edge> nextEdgeList = Iterables.stream(nextEdge.outEdges())
+                    .filter(Edge::isMasterEdge)
```

Comment:
Why is this being added?

Reply:
its a bug. i can submit it separately if needed. Iterables.stream(nextEdge.outEdges() returns both "master" and "reverse" edges -> traverse via "reverse". -> than returns to original edge -> and flagged it as closed. which is incorrect.

Reply:
For tracking purposes, I think this would be better in a separate PR.

Reply:
@Bentleysb This one has also been added. I think the branch is out of date with the dev branch.

Reply:
@Bentleysb This has also been added in a prev PR. I think this branch is out of date with the dev branch.